examples/adaptive_library/BezierCurveRotorPockets.py

```diff
@@ -0,0 +1,305 @@
+"""
+# %%
+# .. note::
+#    This script requires the ``bezier`` Python package to be installed.
+
+# %%
+# This script is designed to be run from a Motor-CAD model based on template "e4a", a 48 slot,
+# 8 pole IPM machine. The model is modified from the template by adjusting the Standard Template
+# geometry parameters as follows:
+#
+# * Set L1 Bridge thickness to 2 mm.
+#
+# * Set L1 Pole V Angle to 180 degrees.
+#
+# * Set L1 Magnet Post to 0 mm.
+#
+# * Set L1 Magnet Separation to 0 mm.
+#
+# * Set L1 Mag Gap Inner to 0 mm.
+
+# %%
+# .. image:: ../../images/Adaptive_Geometry_Bezier_e4a_1.png
+
+# %%
+# If no Motor-CAD file is open, the e4a template will be loaded and the geometry will be adjusted as
+# described above.
+
+# %%
+# Perform required imports
+# ------------------------
+# Import ``pymotorcad`` to access Motor-CAD.
+# Import ``Coordinate``, ``Arc``, ``Line`` ``rt_to_xy``
+# to define the adaptive template geometry.
+# Import ``bezier`` used to draw the curve.
+# Import ``os``, ``shutil``, ``sys``, and ``tempfile``
+# to open and save a temporary .mot file if none is open.
+
+# sphinx_gallery_thumbnail_number = -1
+import os
+import shutil
+import sys
+import tempfile
+
+import bezier
+import numpy as np
+
+import ansys.motorcad.core as pymotorcad
+from ansys.motorcad.core.geometry import Arc, Coordinate, Line, rt_to_xy
+from ansys.motorcad.core.geometry_fitting import return_entity_list
+
+# %%
+# Connect to Motor-CAD
+# --------------------
+# If this script is loaded into the Adaptive Templates file in Motor-CAD, the current Motor-CAD
+# instance is used.
+#
+# If the script is run externally, these actions occur: a new Motor-CAD instance is opened,
+# the e4a IPM motor template is loaded , the geometry changes described above are applied and the
+# file is saved to a temporary folder. To keep a new Motor-CAD instance open after executing the
+# script, use the``MotorCAD(keep_instance_open=True)`` option when opening the new instance.
+# Alternatively, use the ``MotorCAD()`` method, which closes the Motor-CAD instance after the
+# script is executed.
+
+if pymotorcad.is_running_in_internal_scripting():
+    # Use existing Motor-CAD instance if possible
+    mc = pymotorcad.MotorCAD(open_new_instance=False)
+else:
+    mc = pymotorcad.MotorCAD(keep_instance_open=True)
+    # Disable popup messages
+    mc.set_variable("MessageDisplayState", 2)
+    mc.set_visible(True)
+    mc.load_template("e4a")
+
+    # Set Standard Template geometry ready for Adaptive Templates script
+    mc.set_array_variable("BridgeThickness_Array", 0, 2)  # Bridge thickness set to 0
+    mc.set_array_variable("PoleVAngle_Array", 0, 180)  # Pole V angle set to 180
+    mc.set_array_variable("VShapeMagnetPost_Array", 0, 0)  # Magnet Post set to 0
+    mc.set_array_variable("MagnetSeparation_Array", 0, 0)  # Magnet Separation set to 0
+    mc.set_array_variable("VShape_Magnet_ClearanceInner", 0, 0)  # Magnet Inner Gap set to 0
```

Comment:
```suggestion
    mc.set_array_variable("VShapeMagnetPost_Array", 0, 0)  # Magnet post set to 0
    mc.set_array_variable("MagnetSeparation_Array", 0, 0)  # Magnet separation set to 0
    mc.set_array_variable("VShape_Magnet_ClearanceInner", 0, 0)  # Magnet inner gap set to 0
```

Reply:
Keep the same capitalisation used in the Motor-CAD UI, these are proper nouns (names of Motor-CAD parameters in the UI)